src/app/services/offre-de-stage/offre-de-stage.service.ts

```diff
@@ -57,7 +57,7 @@ export class OffreDeStageService {
   }
 
   deleteOffreDeStage(
-    _id: string
+    _id?: string
```

Comment:
On ne veut pas que `_id` soir `undefined` quand on appelle la fonction. 

```suggestion
    _id: string
```